spec/factories/places.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :place do
-    name 'Craftsmen Angers'
+    sequence(:name) { |n| 'Craftsmen Angers #{n}' }
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.